src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,37 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, selectedTab, onTabSelected }) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            return (
+              <li
+                key={tab.id}
+                className={classNames({
+                  'is-active': tab.id === selectedTab.id,
```

Comment:
```suggestion
                  'is-active': isSelectedTab,
```

let's define constant for this `isSelectedTab` and reuse in all places